scripts/twister/alt/zephyr/samples/sensor/bme680/sample.yaml

```diff
@@ -75,3 +75,16 @@ tests:
       - SHIELD=pca63565
       - vpr_launcher_DTC_OVERLAY_FILE="../../../../nrf/boards/shields/pca63565/boards/nrf54l15pdk_nrf54l15_vpr_launcher.overlay"
     platform_allow: nrf54l15pdk/nrf54l15/cpuflpr
+
```

Comment:
The same like for accel_poling, just replace PDK->DK, there is no need for additional configuration.